src/openms/source/SYSTEM/FileWatcher.cpp

```diff
@@ -55,13 +55,13 @@ namespace OpenMS
     //cout << "File changed: " << String(name) << endl;
     // Look up if there is already a timer for this file
     QTimer * timer = nullptr;
-    for (map<QString, QString>::const_iterator it = timers_.begin(); it != timers_.end(); ++it)
+    for (const auto& map : timers_)
```

Comment:
structure binding is more readable
for (const auto& [timer_id, timer_name] : timers_) 

should work